cherrypy/test/sessiondemo.py

```diff
@@ -123,9 +123,11 @@ def page(self):
             'reqcookie': cherrypy.request.cookie.output(),
             'sessiondata': list(cherrypy.session.items()),
             'servertime': (
-                datetime.utcnow().strftime('%Y/%m/%d %H:%M') + ' UTC'
+                datetime.datetime.now(
+                    datetime.UTC).strftime('%Y/%m/%d %H:%M') + ' UTC'
```

Comment:
It occurred to me that "UTC" could be a part of the actual string template. So why not embed it?
```suggestion
                datetime.datetime.now(datetime.UTC).
                strftime('%Y/%m/%d %H:%M UTC')
```

Reply:
done

Reply:
Thanks! Marking this as resolved!